src/types/ErrorMessage.ts

```diff
@@ -0,0 +1,8 @@
+export enum ErrorMessage {
+  FailedLoad = 'Unable to load todos',
+  FailedAddTodo = 'Unable to add a todo',
+  FailedDeleteTodo = 'Unable to delete a todo',
+  FailedUpdateTodo = 'Unable to update a todo',
+  FailedUpdateRequest = 'Unable to update a todo',
```

Comment:
you can delete one
![image](https://github.com/mate-academy/react_todo-app-with-api/assets/91946036/0ddc9de8-b6ea-4738-9e47-cfe81ff973ad)